pkg/reporting/nodes.go

```diff
@@ -83,6 +86,36 @@ func (nri *NodeReportItem) OSVersionPretty() string {
 	return ""
 }
 
+// GetPolicyGroup returns Policy Group or "no group"
+func (nri *NodeReportItem) GetPolicyGroup() string {
+
+	if nri.PolicyGroup == "" {
+		return "no group"
+	}
+
+	return nri.PolicyGroup
+}
+
+// GetPolicy returns Policy Name or "no policy"
+func (nri *NodeReportItem) GetPolicy() string {
+
+	if nri.Policy == "" {
+		return "no policy"
+	}
+
+	return nri.Policy
+}
+
+// GetPolicyWithRev returns formatted Policy name with revision
+func (nri *NodeReportItem) GetPolicyWithRev() string {
+
+	if nri.Policy == "" {
+		return "no policy"
+	}
+
+	return fmt.Sprintf("%s(rev %s)", nri.Policy, nri.PolicyRev)
```

Comment:
```suggestion
	return fmt.Sprintf("%s (rev %s)", nri.Policy, nri.PolicyRev)
```
I think a space makes it more readable

Reply:
done